packages/react-renderer-demo/src/components/notification-panel.js

```diff
@@ -37,38 +37,27 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 const getNotifications = () => {
-  const query = `?orderBy="expired-at"&startAt="${new Date().toISOString()}"&limitToFirst=10`;
-  return fetch(`https://data-driven-forms.firebaseio.com/notifications.json${query}`)
+  return fetch(`/notifications?end=${Date.now()}`)
     .then((data) => data.json())
-    .then((data) => {
-      if (!data) {
-        return [];
-      }
-
-      if (typeof data !== 'object') {
-        return [];
-      }
-
-      if (!Array.isArray(data)) {
-        data = Object.values(data);
-      }
-
-      return data.filter(Boolean).sort((a, b) => b['created-at'].localeCompare(a['created-at']));
-    });
+    .then((data) => data);
 };
 
-const createNotificationId = (notification) => `${notification['created-at']}-${notification['expired-at']}`;
+const createNotificationId = (notification) => notification.activeTill.toString();
 
 const NotificationPanel = ({ isOpen, onClose, anchorRef, setNewMessages }) => {
   const classes = useStyles();
   const [notifications, setNotifications] = useState([]);
   useEffect(() => {
-    getNotifications().then((data = []) => {
-      const lastSeen = JSON.parse(localStorage.getItem('data-driven-forms-last-seen') || '[]');
-      setNewMessages(data.filter((notification) => !lastSeen.includes(createNotificationId(notification))).length);
-      localStorage.setItem('data-driven-forms-last-seen', JSON.stringify(data.map(createNotificationId)));
-      setNotifications(data);
-    });
+    getNotifications()
+      .then((data = []) => {
+        const lastSeen = JSON.parse(localStorage.getItem('data-driven-forms-last-seen') || '[]');
+        setNewMessages(data.filter((notification) => !lastSeen.includes(createNotificationId(notification))).length);
+        localStorage.setItem('data-driven-forms-last-seen', JSON.stringify(data.map(createNotificationId)));
+        setNotifications(data);
+      })
+      .catch(() => {
+        setNotifications([]);
+      });
   }, [setNewMessages]);
 
   return (
```

Comment:
I am sorry, there is actually `created-at` in the old UI code, not `created_at` :-/ So you can change it here.

Reply:
done